### ISSUE TITLE
Thanks for writing these!

### ISSUE THREAD

**Issue by spacerodent** [ryran/loggerclones#1]
No issue. Just wanted to say I really appreciate you writing these.

**Comment by ryran** [comment]
@spacerodent 
Aww you're so welcome Dennis! I'm happy to know that someone (besides the original customer I wrote them for) is getting use out of them.

I am curious: why do you appreciate them? E.g., are you using them? If so, which one and why?

**Comment by spacerodent** [comment]
I was troubleshooting this issue with uwsgi + rsyslog breaking up
messages.  After about an hour of banging my head on the wall I figured out
that logger itself broke up messages. I found a post on a page that
referenced your utilities (I think you might have made the post).  I wat
then able to adjust the rsyslog max message size and do a bunch of testing
and prove rsyslog was working correctly and it was uwsig's syslog plugin
that was breaking up the message.

So your little utility save me a bunch of work AND helped me prove it
wasn't the rsyslog servers fault.

I write the thank you issue because you literally helped finish a really
shitty day on a good note:)

2016-06-30 23:37:24 +0000 Ryan Sawhill Aroha notifications@github.com :
